src/pages/editRecipe/index.jsx

```diff
@@ -0,0 +1,117 @@
+import { useParams } from 'react-router-dom';
+import { Space, message } from 'antd';
+import { useQuery, useMutation } from 'react-query';
+import { useState } from 'react';
+import Title from 'antd/es/typography/Title';
+import axios from 'axios';
+
+import './index.css';
+import BackButton from '../../components/backButton';
+import RecipeForm from '../../components/recipeForm';
+import { LOCAL_API_URL } from '../../constants';
+import { getToken } from '../../helpers';
+
+const authToken = getToken();
+
+function fetchRecipe({ queryKey }) {
+  const id = queryKey[1];
+  return axios.get(`${LOCAL_API_URL}/recipes/${id}`, {
+    params: {
+      populate: '*'
+    }
+  });
+}
+
+function editRecipe({ id, formData }) {
+  return axios.put(`${LOCAL_API_URL}/recipes/${id}`, formData, {
+    headers: {
+      'Content-Type': 'multipart/form-data',
+      Authorization: `Bearer ${authToken}`
+    }
+  });
+}
+
+const EditRecipe = () => {
+  const { id } = useParams();
+  const [messageApi, contextHolder] = message.useMessage();
+  const [recipeData, setRecipeData] = useState([]);
+
+  function updateRecipe(value) {
+    const { name, steps, image } = value.attributes;
+
+    const imageValue = (function () {
+      if (image.data) {
+        const {
+          id,
+          attributes: { name, url }
+        } = image.data;
+        return [{ id, name, url: `http://localhost:1337${url}` }];
```

Comment:
why URL is hard-coded here?

Reply:
actually strapi doesnot provide base url for image.
I have fixed it now

Reply:
![image](https://user-images.githubusercontent.com/114575564/214230543-e57ec823-34e5-4442-93ea-7050d67dc89c.png)